projects/gloo/pkg/plugins/dynamic_forward_proxy/plugin.go

```diff
@@ -128,6 +136,31 @@ func generateCustomDynamicForwardProxyCluster(listenerCfg *dynamic_forward_proxy
 	}
 	return out, nil
 }
+func getCircuitBreakers(listenerCfg *dynamic_forward_proxy.FilterConfig, settings *v1.Settings) *envoy_config_cluster_v3.CircuitBreakers {
+	if listenerCfg.GetCircuitBreakers() != nil {
+		return convertCircuitBreakers(listenerCfg.GetCircuitBreakers())
+	}
+
+	if settings.GetGloo().GetCircuitBreakers() != nil {
+		return convertCircuitBreakers(settings.GetGloo().GetCircuitBreakers())
+	}
+
+	return nil
+}
+
+func convertCircuitBreakers(cfg *v1_circuitbreaker.CircuitBreakerConfig) *envoy_config_cluster_v3.CircuitBreakers {
+	return &envoy_config_cluster_v3.CircuitBreakers{
+		Thresholds: []*envoy_config_cluster_v3.CircuitBreakers_Thresholds{
+			{
+				MaxConnections:     cfg.GetMaxConnections(),
+				MaxPendingRequests: cfg.GetMaxPendingRequests(),
+				MaxRequests:        cfg.GetMaxRequests(),
```

Comment:
If any of these is not specified, will it use the default in envoy?

Reply:
Correct.